flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcTablePlanTest.java

```diff
@@ -70,6 +98,51 @@ public void testFilterPushdown() {
                 "SELECT id, time_col, real_col FROM jdbc WHERE id = 900001 AND time_col <> TIME '11:11:11' OR double_col >= -1000.23");
     }
 
+    /**
+     * Note the join condition is not present in the optimized plan, as it is handled in the JDBC
```

Comment:
Can you log another Jira to improve this, scan source has this ability already.

Reply:
@libenchao I am not sure what I am proposing . I assume this would be a change to the core Flink to include the look up join conditions in the optimised plan. Am I understanding this correctly ?

Reply:
Yes, precisely.

Reply:
> change to the core Flink to include the look up join conditions in the optimised plan

I have raised issue [https://issues.apache.org/jira/browse/FLINK-34170](https://issues.apache.org/jira/browse/FLINK-34170) .

Reply:
@libenchao  just for my understanding ar we thinking that the rowdata sent in on the lookup would contain the pushdown predicates so the code could call and this would handle the predicates and the keys:
` statement = lookupKeyRowConverter.toExternal(keyRow, statement);`
we could then remove the need for?
`statement = setPredicateParams(statement); `

Would you be willing to assign me (and provide minimal guidance if required) FLINK-34170 and review / merge any associated PR?

Reply:
> just for my understanding ar we thinking that the rowdata sent in on the lookup would contain the pushdown predicates so the code could call and this would handle the predicates and the keys:
>  statement = lookupKeyRowConverter.toExternal(keyRow, statement);
> we could then remove the need for?
> statement = setPredicateParams(statement); 

What I was proposing is showing the predicates in the digest of lookup join node. That way, we can see it in the test xml files, and also it can be shown in Flink Web UI.